contents/blog/seed-grow-scale-devrel.mdx

```diff
@@ -58,7 +58,7 @@ PostHog has seen significant growth over the past year resulting in a [Series B
 In the near term, our AAARRRP goals are Activation, Retention, Referral, and Product. Instead of narrowing in on specific activities, we've broadly mapped the priorities as:
 
 1. **Oversee documentation**: This isn't about dictating how documentation is written or being a gatekeeper. It's about encouraging and enabling consistency, having the time to iterate on content that could otherwise become stale, and identifying gaps based on analysis, feedback, or utilizing team experience. DevRel, much like our [amazing design team](/handbook/small-teams/website-docs), can function as a service team with people deployed to the small team that most requires them at that time.
-2. **Catalyze our community**: We have a thriving community across [our GitHub repos](https://github.com/posthog) and the [PostHog community Slack](/slack). On GitHub, we reward contributions with thanks on our READMEs, a listing on the [Contributors page](/contributors) of our website, and credit for our [merch store](https://merch.posthog.com). But we want to do more to build upon this fantastic foundation because [community is core to our business](/handbook/strategy/business-model#why-would-you-work-on-the-community-edition).
+2. **Catalyze our community**: We have a thriving community across [our GitHub repos](https://github.com/posthog) and the [PostHog community](/questions). On GitHub, we reward contributions with thanks on our READMEs, a listing on the [Contributors page](/contributors) of our website, and credit for our [merch store](https://merch.posthog.com). But we want to do more to build upon this fantastic foundation because [community is core to our business](/handbook/strategy/business-model#why-would-you-work-on-the-community-edition).
```

Comment:
```suggestion
2. **Catalyze our community**: We have a thriving community across [our GitHub repos](https://github.com/posthog) and the [PostHog community](/questions). On GitHub, we reward contributions with thanks on our READMEs, a listing on the [Contributors page](/contributors) of our website, and credit for our [merch store](/merch). But we want to do more to build upon this fantastic foundation because [community is core to our business](/handbook/strategy/business-model#why-would-you-work-on-the-community-edition).
```